CONTRIBUTING.md

```diff
@@ -75,6 +73,47 @@ Some companies and organizations are excessively protective with their brands, s
 
 If you are in doubt, feel free to submit it and we'll have a look.
 
+### Assessing Popularity
+
+To be considered popular enough to be within our scope, a brand must be in existence for at least one year (from date of first stable release, where applicable) and meet one of the following metrics of popularity, in order of preference:
+
+1. Its website has a Similarweb global rank of 500k or better.
+    - As Similarweb only update their data once a month, there will be a monitoring window for websites ranked between 450k & 550k until the next update, _unless_ the brand is within scope on any other metric below.
+    - For existing icons in our library, the threshold is dropped from 500k to 750k.
+    - A rank of 2m or worse, without any other metric being provided, will result in the brand being declared outside our scope.
+2. The website's Similarweb rank in any one country is either:
+    - In the top 100, or,
+    - 10k or better, _with_ a global rank of 1m or better.
+3. The website's Similarweb global rank in any one category is either:
+    - In the top 50, or,
+    - 5k or better, _with_ a global rank of 1m or better.
+4. In cases where a brand does not have its own website the Similarweb rank of its parent company will be accepted _if_ the brand is that company's primary product.
+5. Its packages meet one of the following minimum requirements:
+    - [npm](https://www.npmjs.com): 100k weekly downloads,
+    - [JSDelivr](https://www.jsdelivr.com): 1m daily or 35m monthly requests,
+    - [crates.io](https://crates.io): 100k weekly downloads,
+    - [PyPi Stats](https://pypistats.org): 100k weekly downloads, or,
+    - [Packagist](https://packagist.org): 200 average daily installs in a month.
+6. The brand's popularity can be illustrated by other publicly available & verifiable statistic (e.g., downloads, usage).
+    - Stats should preferably also include data on one of our existing brands so a direct comparison can be made.
+7. The brand's popularity can be illustrated through a Google Trends comparison, or similar.
+    - Must be with a similar brand that is already in our library and still within our scope, or that would qualify under any metric.
+    - Must be unambiguous (i.e., it's not a suitable metric for brands with generic words for names).
+    - Trending equal to or higher than the compared brand will be considered in scope.
+    - Trending lower than the compared brand but with an upward trajectory will require the consensus of the person providing the comparison and at least 2 project maintainers.
+8. Where applicable, the primary repository for the brand's GitHub project meets the following requirements (see not below):
+    - A minimum of 5k GitHub stars will be required for consideration, and,
+    - An additional 1k will be required for each year the project is in existence beyond its initial year, providing,
```

Comment:
It has been 11years and 1 months since simple-icons started. So the minimum would be 11.1K+5K=16.1K. Granted our own current status is well above, my personal opinion is that this requirement is a bit harsh.

Reply:
It's _intended_ to be harsh 😉 The whole point of this overhaul is that GH stars are a _terrible_ metric of popularity.

Reply:
I prefer to change the rule, `An additional 1k will be required for each year the project is in existence beyond its initial year, providing` to `Your project should stay active for the long term`.

I agree that there are so many markdown-only projects that get lots of stars. Which is unacceptable.

Reply:
We haven't really solved anything, then; we'd still be accepting projects on 5k GH stars alone. I'd personally prefer to remove that bullet point completely until we can come up with some alternative means of measuring popularity through GitHub.

My reasoning behind the numbers, by the way, was: the 5k was initially decided on as it was half of our (almost) 10k stars at the time and we've gained (just over) 2k stars a year since that, so I went with half of that again.